### DISCUSSION TITLE
Shouldn't 'go' be a dependency in the homebrew package?

### DISCUSSION THREAD

**Discussion by SamAtAnyRoad** [goreleaser/goreleaser#1484] [Ideas, Issue Triage, and General discussions]
I feel crazy even bringing this up, but... goreleaser can't run without go. 

I just tried using goreleaser for the first time, after spotting it in the makefile for a thing I wanted. Homebrew install was painlessly easy. The makefile still failed. Because I didn't have go installed. I installed go. Then everything ran without a hitch.

Since goreleaser needs go to run, it SHOULD have go listed as one of its dependencies, right? Right? Am I crazy? This feels crazy.

**Comment by caarlos0** [comment]
People can install Go in other ways... that's why it isn't there... but maybe it indeed should be...

**Reply by caarlos0** [reply]
https://github.com/goreleaser/goreleaser/pull/1542

**Comment by SamAtAnyRoad** [comment]
Sweet!
This will help folks like me who are touching Go for the first time.
